HIF.md

```diff
@@ -448,6 +450,30 @@ FIXME: define default value
 
 ## Frequency Hopping (FHSS) configuration
 
+Several schemes are used in Wi-SUN to transmit or receive packets using
+frequency hopping. These are independently defined using a set of parameters,
+and may differ between transmission and reception. The following table
+summarizes which commands configure the various FHSS parameters. Note that
+[`REQ_DATA_TX`][tx-req] always needs to be called for transmission, but timing
+parameters are sometimes passed with the command, and sometimes stored in the
+RCP and configured once using a [`SET_FHSS`][fhss] command. See the description
+of these commands for a more detailed explanation.
+
+|  FHSS type                | Reception               | Transmission                |
+|---------------------------|-------------------------|-----------------------------|
+| Unicast to FFN            | [`SET_FHSS_UC`][uc]     | [`REQ_DATA_TX`][tx-req]     |
+| Broadcast to FFN          | [`SET_FHSS_FFN_BC`][bc] | [`REQ_DATA_TX`][tx-req] and [`SET_FHSS_FFN_BC`][bc] |
+| Unicast to LFN            | N/A                     | [`REQ_DATA_TX`][tx-req]     |
+| Unicast PAN Advert to LFN | N/A                     | [`REQ_DATA_TX`][tx-req]     |
+| Broadcast to LFN          | N/A                     | [`SET_FHSS_LFN_BC`][bc-lfn] |
+| Asynchronous              | N/A                     | [`SET_FHSS_ASYNC`][async]   |
```

Comment:
- Maybe it makes sense to repeat "location of parameter" in the columns headers.
 - Does "Reception" of "Unicast to FFN" makes sense?
 
Maybe:
```
|  FHSS type                   | Location of FHSS parameters |
|------------------------------|-----------------------------|
| Rx Unicast                   | [`SET_FHSS_UC`][uc]         |
| Rx Broadcast                 | [`SET_FHSS_FFN_BC`][bc]     |
| Tx Unicast to FFN            | [`REQ_DATA_TX`][tx-req]     |
| Tx Broadcast to FFN          | [`REQ_DATA_TX`][tx-req] and [`SET_FHSS_FFN_BC`][bc] |
| Tx Unicast to LFN            | [`REQ_DATA_TX`][tx-req]     |
| Tx Unicast PAN Advert to LFN | [`REQ_DATA_TX`][tx-req]     |
| Tx Broadcast to LFN          | [`SET_FHSS_LFN_BC`][bc-lfn] |
| Tx Asynchronous              | [`SET_FHSS_ASYNC`][async]   |
```